Code/Correlation.py

```diff
@@ -0,0 +1,40 @@
+from scipy.stats.stats import pearsonr
+from scipy.stats import linregress
+import os
+import numpy 
+from numpy.fft import rfft, rfftfreq
+import librosa
+import matplotlib.pyplot as plt
+import librosa.display
+
+directory = 'C:/Users/BigDDY/Desktop/Test'
+files = os.listdir(directory)
+
+discr = list()
+furie_list = list()
+
+for file in range(0, len(files)):
+    x, sr = librosa.load('C:/Users/BigDDY/Desktop/Test/' + str(files[file]))
+    discr.append(x)
+    
+for file in range(0, len(discr)):
+    furie = rfft(discr[file])
+    furie_list.append(furie)
+
+x, sr = librosa.load('C:/Users/BigDDY/Desktop/File/1.wav')
+sample_furie = rfft(x)
+
+def FindBestCor(sample, listOfFurie):
```

Comment:
add docstring. Example: 
```python
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Parameters
    ----------
    param1 : int
        The first parameter.
    param2 : str
        The second parameter.

    Returns
    -------
    bool
        True if successful, False otherwise.

    """
```